samples/tab-nested-auth/nodejs/README.md

```diff
@@ -110,10 +110,22 @@ The simplest way to run this sample in Teams is to use Microsoft 365 Agents Tool
    
 **This step is specific to Teams:**
 
-- **Edit** the `manifest.json` contained in the  `AppManifest` folder to replace your Microsoft App Id `<<YOUR-MICROSOFT-APP-ID>>` (that was created when you registered your bot earlier) *everywhere* you see the place holder string `<<YOUR-MICROSOFT-APP-ID>>` (depending on the scenario the Microsoft App Id may occur multiple times in the `manifest.json`)
+- **Edit** manifest.json contained in the AppManifest folder to replace the placeholder `{{TEAMS_APP_ID}}` with your actual **Microsoft Teams App Id** (the App Id generated when you registered the app in Azure AD). This placeholder may appear in multiple places.
 
-- **Edit** the `manifest.json` for `{{domain-name}}` with base Url domain. E.g. if you are using ngrok it would be `https://1234.ngrok-free.app` then your domain-name will be `1234.ngrok-free.app` and if you are using dev tunnels then your domain will be like: `12345.devtunnels.ms`.
+- **Edit** the manifest.json to replace `{{AAD_APP_CLIENT_ID}}` with your **Azure AD App Client Id**
```

Comment:
- **Edit** the manifest.json to replace `{{AAD_APP_CLIENT_ID}}` with your Microsoft Entra application ID for the app registered in Azure Active Directory.